source/resources/config/default_config.yml

```diff
@@ -180,68 +38,6 @@ slurm:
         - '.+\.(micro|nano)'   # Not enough memory
         - '.*\.metal'
 
-  # ElasticSearch:
-  # Configure the ElasticSearch/OpenSearch domain used by the slurm cluster
-  # If not specified then won't be created or used by the cluster.
-  # master_nodes: Defaults to 0
-  # data_nodes: Must be a multiple of number_of_azs
-  # ElasticSearch:
-  #   ebs_volume_size: 20
```

Comment:
Why remove these commented out lines? Are they never going to be used? I think it's ok to keep them as a way to reference what can be added.

Reply:
Got it..looks good